src/supervisor/metadata-extractor.ts

```diff
@@ -30,6 +30,7 @@ export function buildImageMetadata(
     delete container.args;
     delete container.env;
     delete container.command;
+    //! would container.envFrom also include sensitive data?
```

Comment:
It shouldn't, it contains only a name to a secret or a configmap - https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envfromsource-v1-core

Reply:
yes, but we also don't need to collect this information as well, right?

Reply:
Right now we don't have use for it, so we indeed don't need to collect it

Reply:
Would we have any use of it in the future? Client's secret has nothing to do with us detecting their vulns?